docs/require_to_webpack.rst

```diff
@@ -0,0 +1,66 @@
+Converting from Require to Webpack
+==================================
+
+This guide will help you convert a legacy Require module include to
+Webpack. Legacy require modules make use of the mako ``requirejs``
+block, which looks like this:
+
+::
+
+    <%block name="requirejs">
+        require(["js/factories/myawesomefactory"], function(MyAwesomeFactory) {
+            MyAwesomeFactory(${awesome_data | n, dump_js_escaped_json});
+        });
+    </%block>
+
+This example assumes the following: - You’re working in Studio (no
+attempts have been made yet to convert Require to Webpack in LMS).
+Studio’s root JS directory is ``/cms/static/js``. - A factory
+MyAwesomeFactory invoked from within a Mako template - MyAwesomeFactory
+lives in ``/cms/static/js/factories/myawesomefactory.js`` - There’s a
+contextual Python variable ``awesome_data`` used to populate
+``MyAwesomeFactory``.
```

Comment:
Is this supposed to be an unordered list? I think this got collapsed into one paragraph.

Reply:
Ooops -- that's what I get for writing in Markdown and using an online converter 🙄